core/test/unit/services/url/Resources_spec.js

```diff
@@ -108,13 +108,13 @@ describe('Unit: services/url/Resources', function () {
 
                 should.exist(resources.getByIdAndType(options.eventData.type, options.eventData.id));
                 obj.tags.length.should.eql(1);
-                Object.keys(obj.tags[0]).sort().should.eql(['id', 'slug'].sort());
+                Object.keys(obj.tags[0]).sort().should.eql(['id', 'post_id', 'slug', 'visibility'].sort());
```

Comment:
`withRelatedFields` doesn't contain these fields, will look into the code itself to confirm why they are being returned here

Reply:
After a little investigation, looks like there is a bug in the current implementation of `raw_knex.fetchAll` in the base model where these fields are not properly filtered on related tables. @kirrg001 I propose fixing this in a separate issue to be able to merge these changes sooner?